testdata/vcap_services.json

```
@@ -0,0 +1,44 @@
+{
+    "elephantsql": [
+      {
+        "name": "elephantsql-binding-c6c60",
+        "binding_guid": "44ceb72f-100b-4f50-87a2-7809c8b42b8d",
+        "binding_name": "elephantsql-binding-c6c60",
+        "instance_guid": "391308e8-8586-4c42-b464-c7831aa2ad22",
+        "instance_name": "elephantsql-c6c60",
+        "label": "elephantsql",
+        "tags": [
+          "postgres",
+          "postgresql",
+          "relational"
+        ],
+        "plan": "turtle",
+        "credentials": {
+          "uri": "postgres://exampleuser:examplepass@babar.elephantsql.com:5432/exampleuser"
```

```suggestion
          "uri": "postgres://exampleuser:examplepass@postgres.example.com:5432/exampleuser"
```

Use example.com domain for tests/documentation instead of a real domain.

---

I just copied the example from the cloudfoundry documentation, but that is a fair point.